.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 package-lock.json
+AWSconfig.js
 bundle.js
```

Comment:
Somehow this still got committed and pushed, maybe try removing the file from the branch. You probably also want to add .DS_Store to this as well